content/install_chef_air_gap.md

```diff
@@ -161,9 +161,11 @@ custom template, and then modify the package and Ruby gem sources.
     template file name, be sure to replace `airgap.erb` with the
     template file you created during the last step.
 
-4. Update `airgap.erb` to replace `omnitruck.chef.io` with the URL of
+4. Update `airgap.erb` to replace `https://chefdownload-commericial.chef.io/install.sh?license_id=<LICENSE_ID>` with the URL of
     `install.sh` on your artifact store:
 
+    {{< note >}}Please follow steps mentioned in this document [Chef Software Install Script](/chef_install_script/). Please replace `<LICENSE_ID>` with your licenseId..{{< /note >}}
+
     ```ruby
```

Comment:
I don't think this is correct. The `airgap.erb` file is generated from a template and unless that template has been updated (maybe it has?) the correct URL here is `omnitruck.chef.io`. This change  also doesn't make sense in this context since they're replacing the omnitruck URL with the URL of their own deployment.

@clintoncwolfe @sanghinitin

Reply:
It would seem in this context the user is looking for the omnitruck.chef.io value and inserting a per-org custom value. So the correct value would be `omnitruck.chef.io` i would think.

Reply:
@clintoncwolfe,
Initially, the product install scripts were downloaded using the Omnitruck service only and therefore the value used was `omnitruck.chef.io`. But now, we have upgraded the Download APIs and added the support for the install scripts, and moving ahead these scripts will be available via the Download APIs only.
Therefore, we have added this change and updated the URLs from `omnitruck.chef.io` to the Downloads URL

Request you to please do suggest the correct value that should be used in this case

Reply:
@sandhi18 This is a different, special circumstance. Here, the customer is setting up an airgap environment with a customized download URL that only the customer will know. The value that will appear in the file that they need to replace is `omnitruck.chef.io` and they need to replace it with their customer value. You could say this:
```
Update `airgap.erb` to replace `https://chefdownload-commericial.chef.io/install.sh?license_id=<LICENSE_ID>` with the URL of
    `install.sh` on your artifact store:
    YOUR_CUSTOM_DOWNLOAD_URL
```

Reply:
Yeah, but this assumes that the value in this airgab.erb file is `https://chefdownload-commericial.chef.io/install.sh?license_id=<LICENSE_ID>`, when it's actually `omnitruck.chef.io`. I ran the procedure described in this doc locally and it's `omnitruck.chef.io`, so unless someone has a fix in to update this, then this isn't going to be different. And if it is different, then it would likely be just the  FQDN `chefdownload-commericial.chef.io`.

Reply:
@IanMadd @clintoncwolfe 
As per my understanding from the comment, it looks like the customer will be using some custom URLs so, as suggested by you, we have removed all the changes that were made regarding the install scripts in this particular file and moved it back to `omnitruck.chef.io`.
Request you to please review the changes and let me know if these look good
CC: @sonarora